modules/appendix/examples/friend_net/select_statement_index_order_by_descending.gsql

```diff
@@ -3,7 +3,7 @@ CREATE QUERY top_popular() FOR GRAPH Friend_Net {
 	SumAccum<INT> @num_coworkers;
 	start = {Person.*};
 
-	result = SELECT v FROM start -((Friend|Coworker):e)- Person:v
+	result = SELECT v FROM (start) -[e:Friend|Coworker]- (v:Person)
```

Comment:
```suggestion
	result = SELECT v FROM (:start) -[e:Friend|Coworker]- (v:Person)
```
Note the colon before start.
`start` is a vertex set; vertex sets always come after a colon.